Makefile

```diff
@@ -49,17 +56,10 @@ vet: ## Run go vet against code.
 verify-no-changes: ## verify there are no un-staged changes
 	./hack/verify-diff.sh
 
-.PHONY:vendor
```

Comment:
Is vendoring happens in a different job now ?

Reply:
I removed vendoring completely, doesn't make sense for a library. See PR description and commit messages. The using component vendors everything.